### PR TITLE
🪟 🎨 Update favicon and table row image styles

### PR DESCRIPTION
**Pull request by teallarson**
## What
Changes driven by weekly Design/Front End Bug Sync.  Two action items from today's meeting:
1. Update favicon
2. Update the style of in-row number badges (image below shows new design):
Screen Shot 2022-06-22 at 11 21 30 AM

## How
1. Swapped image file 
2. Converted `ImageBlock` to use .scss modules and classnames and updated to match @Upmitt's designs

## Recommended reading order
1. `favicon.ico`
2. Others

## Note:
Other icon sizes in the app will be updated when Nico completes them.  For now, just updating the favicon is ok per his recommendation.

### CODE REVIEW

File: airbyte-webapp/src/components/ImageBlock/ImageBlock.tsx [hunk 1]
```diff
@@ -1,39 +1,29 @@
...
+  });
+  return (
+    <div className={imageCircleClassnames}>
+      {num ? <div className={styles.number}>{num}</div> : <div className={styles.icon}>{getIcon(img)}</div>}
+    </div>
+  );
+};
 
 export default ImageBlock;
 export { ImageBlock };
```
Comment (edmundito) [inline airbyte-webapp/src/components/ImageBlock/ImageBlock.tsx:29]:
```suggestion
```

**Review by edmundito — COMMENTED** [review]
Changes look good, just had a nit and one request: Would it be possible to add the component to storybook?

File: airbyte-webapp/src/components/ImageBlock/ImageBlock.module.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  border-radius: 0%;
+}
+
+.icon {
+  max-height: 100%;
+  max-width: 100%;
+}
+
+.number {
+  font-family: "Inter";
```
Comment (dizel852) [inline airbyte-webapp/src/components/ImageBlock/ImageBlock.module.scss:35]:
```suggestion
  font-family: fonts.$regularFont;
```

File: airbyte-webapp/src/components/ImageBlock/ImageBlock.module.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
+@use "../../scss/colors";
```
Comment (dizel852) [inline airbyte-webapp/src/components/ImageBlock/ImageBlock.module.scss:1]:
```suggestion
@use "../../scss/colors";
@use "../../scss/fonts";
```

**Comment by tealjulia** [comment]
Will add to Storybook!  Thanks for catching the pieces I missed there with exporting/variables.

**Review by dizel852 — APPROVED** [review]

**Review by edmundito — APPROVED** [review]

**Pull request merged**